packages/config/config/devices/0x0403/shha10000.json

```
@@ -16,6 +16,240 @@
 		"min": "0.0",
 		"max": "255.255"
 	},
+	"paramInformation": {
+		"1": {
+			"label": "Over current protection",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disabled",
+					"value": 0
+				},
+				{
+					"label": "Enabled",
+					"value": 1
+				}
+			]
+		},
+		"2": {
+			"label": "Device Status After Power Failure",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 2,
+			"defaultValue": 0,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "State before failure",
+					"value": 0
+				},
+				{
+					"label": "On",
+					"value": 1
+				},
+				{
+					"label": "Off",
+					"value": 2
+				}
+			]
+		},
+		"3": {
+			"label": "Notification Load Status Change",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 2,
+			"defaultValue": 1,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disabled",
+					"value": 0
+				},
+				{
+					"label": "Send report",
+					"value": 1
+				},
+				{
+					"label": "Send report only with manual change",
+					"value": 2
+				}
+			]
+		},
+		"4": {
+			"label": "LED Indicator Mode",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "LED is permanently on when device on",
+					"value": 0
+				},
+				{
+					"label": "LED is on for 5 seconds when device changing to on",
+					"value": 1
+				}
+			]
+		},
+		"16": {
+			"label": "Threshold Power Report Value",
+			"description": "The threshold value serves as an additional condition of the time-based report",
+			"valueSize": 2,
+			"unit": "Watt",
+			"minValue": 0,
+			"maxValue": 65535,
+			"defaultValue": 50,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true,
+			"options": [
+				{
+					"label": "Disabled",
+					"value": 0
+				}
+			]
+		},
+		"17": {
+			"label": "Threshold Power Report Percentage",
+			"description": "The threshold value serves as an additional condition of the time-based report",
+			"valueSize": 1,
+			"unit": "%",
+			"minValue": 0,
+			"maxValue": 127,
+			"defaultValue": 10,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true,
+			"options": [
+				{
+					"label": "Disabled",
+					"value": 0
+				}
+			]
+		},
+		"100": {
+			"label": "Reset Parameter 102-104",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 85,
+			"defaultValue": 0,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disabled",
+					"value": 0
+				},
+				{
+					"label": "Reset to default Values",
+					"value": 85
+				}
+			]
+		},
+		"101": {
+			"label": "Power Report (Watt)",
```

I think this should have "Interval" in the label, maybe like this
```suggestion
			"label": "Power (Watt) Report Interval",
```

---

or (Watt) at the end. Same goes for the next params.

---

Completed